ophyd/sim.py

```diff
@@ -902,9 +905,13 @@ def _scan(self):
             with self._lock:
                 self._data.append(event)
 
-    def stop(self, *, success=False):
+    def stop(self, success=False):
         pass
 
+    # Placeholder required for components to conform to the Bluesky Moveable protocol
+    def set(self, *args, **kwargs) -> Any:
+        return super().set(*args, **kwargs)
+
```

Comment:
Perhaps this should just pass as well? This is a sim so wouldn't have a `super()` anyway right?

Reply:
Correction, it should still return a `NullStatus`.